docs/NSR.md

```diff
@@ -17,21 +17,21 @@ a.	Segment Certificate Systems into networks based on their functional or logica
 
 b.	Apply equivalent security controls to all systems co-located in the same network with a Certificate System;
 
-c.	Maintain Root CA Systems in a High Security Zone and in an offline state or air-gapped from all other networks;
+c.	Maintain Root CA Systems in a High Security Zone and in an offline state or air-gapped from all other networks. Offline Root CA systems shall be a non-networked, stand-alone system that is powered down when not in use; 
```

Comment:
The existing definition may be enough, but look at changes if they are changed by removing this line.

Reply:
See my comment below.  If we define those terms, I don't know that it matters if they remove that line.